src/datadog/grok/parse_grok.rs

```diff
@@ -51,19 +51,15 @@ fn apply_grok_rule(source: &str, grok_rule: &GrokRule) -> Result<Value, Error> {
                     filters,
                 }) = grok_rule.fields.get(name)
                 {
-                    filters.iter().for_each(|filter| {
-                    if let Some(ref mut v) = value {
-                        value = match apply_filter(v, filter) {
-                            Ok(Value::Null) => None,
-                            Ok(v) if v.is_object() => Some(parse_keys_as_path(v)),
-                            Ok(v) => Some(v),
-                            Err(error) => {
-                                warn!(message = "Error applying filter", field = %field, filter = %filter, %error);
```

Comment:
Did we lose this warning when moving to the use of `?`? I don't see it in changes to callers.

Reply:
Oh I could call that out. The removal of the warning log was intentional. `parse_groks` stdlib function catches errors and logs them so when we change the behavior here to surface the error, this warning log is redundant.